FreetimeTests/BookmarkStoreTests.swift

```diff
@@ -56,4 +56,12 @@ final class BookmarkStoreTests: XCTestCase {
 
         XCTAssert(BookmarksStore.shared.bookmarks.count == 0)
     }
+    
+    func test_containsBookmark() {
+        let b1 = BookmarkModel(type: .repo, name: "GitHawk", owner: "rizwankce")
+        
+        BookmarksStore.shared.add(bookmark: b1)
```

Comment:
Gotta get rid of this singleton 😉

Reply:
Yes Yes. I just don't want to give a big PR with all changes. Wanted to give a separate/small PR after I fix the initial round of bug fixes. So it would be easy for us to review/conflicts free PR.